submit-web/src/components/Submission/SubmissionItemTableRow/ProponentSubmissionItemTableRow.tsx

```diff
@@ -133,7 +133,7 @@ export default function ProponentSubmissionItemTableRow({
               submissionPackage?.submitted_on &&
               submissionPackage.update_requests.filter(
                 (updateRequest) =>
-                  updateRequest.status === UPDATE_REQUEST_STATUS.OPEN.value,
```

Comment:
I think entities want to only see Open requests

Reply:
Thanks. The ticket said that entity should be able to submit unless the update request is accepted so I made this change so that they can edit the data unless the request is accepted by staff.